src/pattern-library/components/Library.js

```diff
@@ -81,38 +85,69 @@ function PageIntro({ children }) {
 }
 
 /**
- * Render info about a single pattern (or component) on a pattern-library page.
+ * Render info about a primary section of a page
  *
- * @param {LibraryBaseProps} props
+ * @param {object} props
+ *   @param {Children} props.children
+ *   @param {string} [props.id]
+ *   @param {Children} [props.intro]
+ *   @param {string} props.title
  */
-function Pattern({ children, title }) {
+function Section({ children, id, intro, title }) {
   return (
-    <section className="space-y-8">
-      <h2 className="text-2xl text-slate-7">{title}</h2>
-      <div className="space-y-8 px-4">{children}</div>
+    <section className="pb-16 space-y-8">
+      <h2 className="text-3xl font-bold" id={id}>
+        {title}
+      </h2>
+      {intro && <SectionIntro>{intro}</SectionIntro>}
+      <div className="space-y-16 styled-text">{children}</div>
     </section>
   );
 }
 
 /**
- * @typedef LibraryExampleProps
- * @prop {import("preact").ComponentChildren} [children]
- * @prop {string} [title]
- * @prop {'split'|'wide'} [variant='split'] - Layout variant. Applies
- *   appropriate className.
- *   - Split (default) lays out in a row. Non-demo example content is rendered
- *     left, with demos right. Demos in this variant stack vertically.
- *   - Wide lays out in a full-width column. Non-example is rendered first,
- *     then a row to contain demos. Demos in this variant render next to each
- *     other in a single row.
+ * Page introductory text
+ *
+ *  @param {object} props
+ *    @param {Children} props.children
  */
+function SectionIntro({ children }) {
+  return (
+    <div className="styled-text text-base space-y-3 leading-relaxed">
+      {children}
+    </div>
+  );
+}
 
 /**
- * Render example content and optional Demo(s) for a pattern.
+ * Render info about a secondary section of a page
  *
- * @param {LibraryExampleProps} props
+ * @param {object} props
+ *   @param {Children} props.children
+ *   @param {string} [props.id]
+ *   @param {Children} [props.intro]
+ *   @param {string} props.title
  */
-function Example({ children, title, variant = 'split' }) {
+function Pattern({ children, id, title }) {
+  return (
+    <section className="space-y-8">
+      <h3 className="text-2xl text-slate-7" id={id}>
+        {title}
+      </h3>
+      <div className="space-y-8 px-4">{children}</div>
+    </section>
+  );
+}
+
+/**
+ * Render information about a tertiary section on a page.
```

Comment:
This component seems more specific than just "a tertiary section on a page".

Reply:
(You're right. I expanded this)